app/models/basket.rb

```diff
@@ -12,4 +14,27 @@ def build_entries_from_file_upload(file)
       entry.save
     end
   end
+
+  private
+
+  def update_taxes_update_total
+    self.sales_taxes = sum_tax
+    self.total = sum_total
+  end
+
+  def sum_tax
+    sum = 0
+    entries.each do |entry|
+      sum += entry.basic_tax.to_f + entry.import_tax.to_f
```

Comment:
Try not to use `float` when using money because in some cases they can be imprecise. More info here: https://www.honeybadger.io/blog/ruby-currency/